vn/data/tables/vs3-sct.tbm

```diff
@@ -181,19 +175,19 @@ mn.LuaSEXPs["lua-vn-load"].Action = function(filename, start, hideCursor)
 
 end
 
-function VS3:L(filename, start, stayInGame)
-
-	self:LoadScene{filename=filename .. ".txt", start=start, stayInGame=stayInGame}
-	
-end
-
```

Comment:
Why remove this?
I get that it's mostly a remnant of calling the VN-script from script-eval blocks with short line length, but still.

Reply:
It wasn't referenced elsewhere in the script file, and I hope we can assume that version 3 of the VN script won't be invoked via script-eval.  So to me it was just clutter.  Given the length and complexity of the script, I'm in favor of making it simpler in any way possible.